README.md

```diff
@@ -1,18 +1,51 @@
 
 ## Welcome to the Gluu Flex Project
 
+Gluu Flex is a commercial distribution of the Linux Foundation 
+[Janssen Project](https://jans.io). 
+It was created for businesses who want the protections of a commercial license, 
+plus some additional tools convenient for enterprise scale. 
+Flex is built from Janssen Project components,  so migrating to or from the 
+open source distribution is easy–just update your configuration.
```

Comment:
fix easy-just typo. Its actually a simple two way pathway. Once you have flex you can move back with unsubscribing to the admin UI and any support contract.